### PR TITLE
Return list of users in `team list` command for a non admin users

### PR DESCRIPTION
**Pull request by drgarcia1986**
This change is [Reviewable](https://reviewable.io/reviews/luizalabs/teresa-api/237)

### CODE REVIEW

File: pkg/server/team/team.go [hunk 1]
```diff
@@ -95,9 +90,25 @@ func (dbt *DatabaseOperations) ListByUser(userEmail string) ([]*storage.Team, er
...
 		)
 	}
+
+	if err = dbt.associateUsersWithTeams(teams); err != nil {
+		return nil, err
+	}
 	return teams, nil
 }
 
+func (dbt *DatabaseOperations) associateUsersWithTeams(teams []*storage.Team) error {
```
Comment (arnaldopereira) [inline pkg/server/team/team.go]: I don't like this function name. It associates users with teams and perform a find operation, so it seems it goes a _little_ beyond the association.

Comment (drgarcia1986) [inline pkg/server/team/team.go]: hm you're right @arnaldopereira .
What about `findUsersOfTeams` ?

Comment (aguerra) [inline pkg/server/team/team.go]: Maybe `findTeamUsers`?

Comment (drgarcia1986) [inline pkg/server/team/team.go]: ☝️ LGTM

Comment (arnaldopereira) [inline pkg/server/team/team.go]: LGTM

**Comment by drgarcia1986** [comment]
updated @aguerra @arnaldopereira

**Pull request merged**